plugin/completion/lib_complete.py

```diff
@@ -143,7 +147,8 @@ def parse_tu(self, view):
                     args=self.clang_flags,
                     unsaved_files=files,
                     options=TU.PARSE_PRECOMPILED_PREAMBLE |
-                    TU.PARSE_CACHE_COMPLETION_RESULTS)
+                    TU.PARSE_CACHE_COMPLETION_RESULTS |
+                    TU.PARSE_INCLUDE_BRIEF_COMMENTS_IN_CODE_COMPLETION)
```

Comment:
Ok, I think I was just being dumb. I see the comments now. Guess I just tried on the non-documented function in my codebase :stuck_out_tongue_closed_eyes: